### PR TITLE
Add some scripts for cookbook branch testing

### PR DESCRIPTION
**Pull request by meatballhat**

### CODE REVIEW

**Comment by meatballhat** [comment]
This is working as expected, e.g.:

``` bash
bundle exec ./bin/packer-build-cookbooks-branch meat-intern-git ci-connie.yml
```

results in: https://travis-ci.org/travis-infrastructure/packer-build/jobs/177561367/config

**Comment by igorwwwwwwwwwwwwwwwwwwww** [comment]
@meatballhat this is for trigger a build manually, right?

**Comment by meatballhat** [comment]
@igorwwwwwwwwwwwwwwwwwwww confirm, so that creation of a PR with a hardcoded branch is no longer necessary.  I've been meaning to do this for months.  It's one of the reasons why the packer templates accept the branch names from the env.

File: bin/packer-build-trigger [hunk 1]
```diff
@@ -14,7 +14,16 @@ BODY_TMPL=${TOP}/.packer-build-pull-request-${TRAVIS_PULL_REQUEST}-tmpl.yml
...
+if [[ "${TRAVIS_COOKBOOKS_TEST_BRANCH}" ]]; then
+  BODY_TMPL_TMP="$(mktemp "${TMPDIR:-/var/tmp}/packer-build-trigger.yml.XXXXXXX")"
+  trap 'rm -f '${BODY_TMPL_TMP} EXIT INT TERM QUIT
+  "${TOP}/bin/body-tmpl-with-cookbooks-branch" \
+    "${BODY_TMPL}" "${TRAVIS_COOKBOOKS_TEST_BRANCH}" >"${BODY_TMPL_TMP}"
+  BODY_TMPL="${BODY_TMPL_TMP}"
+fi
+
 set -o xtrace
+export DEBUG=1
```
Comment (meatballhat) [inline bin/packer-build-trigger]: whoops, should back this out

**Pull request merged**